src/oom/oomd-manager.c

```diff
@@ -374,8 +374,17 @@ static int monitor_swap_contexts_handler(sd_event_source *s, uint64_t usec, void
         return 0;
 }
 
+static void clear_candidate_hashmap(Manager **m) {
+        if (m && *m && (*m)->monitored_mem_pressure_cgroup_contexts_candidates)
+                hashmap_clear((*m)->monitored_mem_pressure_cgroup_contexts_candidates);
+}
```

Comment:
Please call this with "p" at the end:
```C
static void clear_candidate_hashmapp(Manager **m) {
     if (*m)
           hashmap_clear((*m)->monitored_mem_pressure_cgroup_contexts_candidates);
}
```
(`m` can never be null, and hashmap_clear() checks for NULL internally too.)